index.html

```diff
@@ -24,52 +24,56 @@ <h1 class="pl-heading pl-element-title">{{ post.title }}</h1>
                 </ul>
```

Comment:
Reminder: whatever layout changes are done here should also be done to the permalink ``layouts/element.html`` template as well.